odoo-sentinel-shell.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# force to xterm because of bogus terminal emulator clients
+export TERM=xterm
+exec odoo-sentinel
```

Comment:
@guewen not sure how to check this in the PR : is this file executable?

Reply:
It seems we can't from github.

It is:
```
-rwxrwxr-x 1 gbaconnier gbaconnier  110 mai   24 09:14 odoo-sentinel-shell.sh
```